exonum-java-binding-core/rust/src/utils/mod.rs

```diff
@@ -24,8 +24,10 @@ mod pair_iter;
 mod resource_manager;
 
 pub use self::conversion::{convert_hash, convert_to_hash, convert_to_string};
-pub use self::errors::{check_error_on_exception, get_and_clear_java_exception, panic_on_exception,
-                       unwrap_jni, unwrap_jni_verbose};
+pub use self::errors::{
+    check_error_on_exception, get_and_clear_java_exception, panic_on_exception, unwrap_jni,
```

Comment:
Why do these changes appear in every PR? Haven't we merged them?

Reply:
We updated rustfmt to the latest version, but a week later next version was released which slightly changed syntax for imports. I suppose no commit with fix were merged yet